src/data-sources/DataSourceConnector.ts

```diff
@@ -97,6 +97,25 @@ export class DataSourceConnector {
         return;
       }
 
+      // Checking if this is a config value
+      if (dependency.startsWith('connection:')) {
+        const connection = dependency.substr(dependency.indexOf(':') + 1);
+        const ds: IDataSource = this.dataSources[Object.keys(this.dataSources)[0]];
+        if ( !ds.plugin.hasOwnProperty('connections')) {
```

Comment:
I just noticed - you are using `this` in a static method. I don't know if/why this works, but you should use `DataSourceConnector.datasources` unless I'm missing something.